tests/attr/test_deeplift_basic.py

```diff
@@ -199,3 +261,35 @@ def _deeplift_assert(self, model, attr_method, inputs, baselines):
                 ig = IntegratedGradients(model)
                 attributions_ig = ig.attribute(inputs, baselines)
                 assertAttributionComparision(self, attributions, attributions_ig)
+
+
+def _hypothetical_contrib_func(multipliers, inputs, baselines):
```

Comment:
The expanded/repeated inputs are passed here for the SHAP version but we can potentially pass original inputs and baselines to the function.
cc: @AvantiShri

Reply:
Passing the expanded inputs sounds fine to me and is also much cleaner from an API perspective! Thank you!